modules/caddyhttp/reverseproxy/fastcgi/client_test.go

```diff
@@ -135,16 +134,19 @@ func sendFcgi(reqType int, fcgiParams map[string]string, data []byte, posts map[
 			length = len(data)
 			rd := bytes.NewReader(data)
 			resp, err = fcgi.Post(fcgiParams, "", "", rd, int64(rd.Len()))
+			resp.Body.Close()
 		} else if len(posts) > 0 {
 			values := url.Values{}
 			for k, v := range posts {
 				values.Set(k, v)
 				length += len(k) + 2 + len(v)
 			}
 			resp, err = fcgi.PostForm(fcgiParams, values)
+			resp.Body.Close()
```

Comment:
You might want to wait until the `Body` is read below on line 170 before closing it, no ?